### DISCUSSION TITLE
nvrtc: error: invalid value for --gpu-architecture (-arch)

### DISCUSSION THREAD

**Discussion by fresh382227905** [apache/incubator-mxnet#20136] [Q&A]
3070 graphics card
cuda 11.0 
mxnet 1.8
running the following code, It works well without bugs.
 `import mxnet as mx
a = mx.nd.ones((2, 3))
b = a * 2 + 1
b.asnumpy()
array([[ 3.,  3.,  3.],
       [ 3.,  3.,  3.]], dtype=float32)`
However, when run another code , it appears the following errors:
'
mxnet.base.MXNetError: Traceback (most recent call last):
  File "../src/operator/fusion/fused_op.cu", line 647
MXNetError: Check failed: compileResult == NVRTC_SUCCESS (5 vs. 0) : NVRTC Compilation failed. Please set environment variable MXNET_USE_FUSION to 0.
nvrtc: error: invalid value for --gpu-architecture (-arch)
'
I'm not sure where the trouble is. Can you give me some advice? 
Thanks!

**Comment by szha** [comment]
I think people will need to know about the code that produce the error to be able to help. cc @ptrendx who may have general comment.

**Comment by ptrendx** [comment]
RTX 3070 is GA102 (sm_86) architecture, which is not supported by CUDA 11.0 (the first CUDA version that supports it is 11.1), please use CUDA 11.2 version (mxnet-cu112) with this card.
